src/command.rs

```diff
@@ -328,7 +328,7 @@ pub fn run() -> Result<()> {
 			)?;
 
 			let state_version = Cli::native_runtime_version(&chain_spec).state_version();
-			let block: Block = generate_genesis_block(&chain_spec, state_version)?;
+			let block: Block = generate_genesis_block(&**chain_spec, state_version)?;
```

Comment:
This dereference seems weird, doesnt it work with only one `&*chain_spec` if we change the above chain_spec declaration to:
```
let chain_spec = load_spec(
```